src/instructlab/model/accelerated_train.py

```diff
@@ -236,22 +283,19 @@ def _run_phased_training(
             journal.journal.train_1 = phase_model
         journal.commit()
 
-        _training_phase(
-            train_args=train_args.model_copy(deep=True),
-            torch_args=torch_args,
-            data_path=phase1_data,
-            checkpoint_dir=phase1_checkpoints_dir,
-            num_epochs=phase1_num_epochs,
-            samples_per_save=phase1_samples_per_save,
-            effective_batch_size=phased_phase1_effective_batch_size,
-            # model override not necessary because we expect model to come from ctx.params.model_path.
+        _run_phase(
```

Comment:
Could you please use arg names rather than positional?
e.g.
```suggestion
        _run_phase(train_args=train_args, torch_args=torch_args, data_path=phase1_data, checkpoint_dir=phase1_checkpoints_dir, ...)
```